### PR TITLE
Bengali translation of the PHP 8 Announcement Page

### PR DESCRIPTION
**Pull request by SMAtaurRahman**

### CODE REVIEW

File: releases/8.0/bn.php [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+include_once __DIR__ . '/common.php';
+
+releases\php80\language_redirect('bn');
+
+releases\php80\common_header(
+    'PHP 8.0 is a major update of the PHP language. ' .
+    'It contains many new features and optimizations including ' .
+    'named arguments, union types, attributes, constructor property promotion, ' .
+    'match expression, nullsafe operator, JIT, and ' .
+    'improvements in the type system, error handling, and consistency.');
```
Comment (cmb69) [inline releases/8.0/bn.php:12]: This paragraph should be translated as well.

File: releases/8.0/bn.php [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+      <div class="php8-compare__arrow"></div>
+      <div class="php8-compare__block example-contents">
+        <div class="php8-compare__label php8-compare__label_new">PHP 8</div>
+        <div class="php8-code phpcode">
+            <?php highlight_php_trimmed(
+                'echo match (8.0) {
+  \'8.0\' => "Oh no!",
+  8.0 => "This is what I expected",
+};
+//> This is what I expected'
```
Comment (cmb69) [inline releases/8.0/bn.php:241]: Consider to translate the strings ("Oh no!" and "This is what I expected") in this example.

File: include/languages.inc [hunk 3]
```diff
@@ -13,6 +13,7 @@ $LANGUAGES = array(
     'en'    => 'English',
     'ar'    => 'Arabic',
     'bg'    => 'Bulgarian',
+    'bn'    => 'Bengali',
```
Comment (cmb69) [inline include/languages.inc:16]: Please revert this change. It is no longer needed; the addition in releases/8.0/common.php is sufficient.

**Review by cmb69 — COMMENTED** [review]
Thank you for the translation! I can't comment on the translated parts, because I don't speak Bengali, but left a few notes.

File: releases/8.0/bn.php (hunk 1 above)
Comment (SMAtaurRahman) [reply releases/8.0/bn.php]: Thank you for pointing it out. I have updated the translation.

File: releases/8.0/bn.php (hunk 2 above)
Comment (SMAtaurRahman) [reply releases/8.0/bn.php:241]: Usually, we don't write bengali in code comments. Because some editor may not have unicode support. IMO, I think we should left it as it is. Some of the other translation left it too.

File: include/languages.inc (hunk 3 above)
Comment (SMAtaurRahman) [reply include/languages.inc]: Thanks. Should I rebase on master?

File: releases/8.0/bn.php (hunk 2 above)
Comment (cmb69) [reply releases/8.0/bn.php:241]: Fair enough. :)

File: include/languages.inc (hunk 3 above)
Comment (cmb69) [reply include/languages.inc]: Unless there would be merge conflict, rebase is not necessary.

**Comment by cmb69** [comment]
Could someone please review?
